src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -410,7 +430,7 @@ public SubJobWorker(MultiJobProject multiJobProject,
             this.queue = queue;
         }
 
-        public void run() {
+        public Object call() {

```

Comment:
Makes sense to add Javadoc that the return value is useless. On the other hand, support of meaningful result object may be useful in the future (e.g. <code>SubJobCallResult</code>)